tests/plugins/autolink/autolink.js

```diff
@@ -99,6 +99,14 @@
 			assertPasteEvent( this.editors.classic, { dataValue: pastedText }, { dataValue: pastedText, type: 'html' } );
 		},
 
+		// (#4858)
+		'test URL link with encoded characters': function() {
+			var pastedText = 'https://www.google.com/test/?one=one&amp;two=two&amp;three',
```

Comment:
Shouldn't it contain `&`? Probably this test passes due to #4912.

Reply:
No, `assertPasteEvent` is not escaping characters by itself as it skips some parts of our data processing, so we have to feed it with correct input. Anyway, I've added an additional check confirming that the test is valid.